.github/workflows/lint.yml

```diff
@@ -16,5 +16,5 @@ jobs:
           node-version: 16.x
           cache: npm
       - run: yarn install --frozen-lockfile
-      - run: yarn run lint
+      - run: yarn run lint --disable MD013
```

Comment:
[optional] it may be beneficial to introduce a root-level `.markdownlint.yaml` like https://github.com/celestiaorg/celestia-app/blob/main/.markdownlint.yaml so that developers who run `yarn run lint` locally have the same rules applied as the ones running in CI.

Reply:
great suggestion, thanks @rootulp!

Reply:
made an issue for this, too: #457